hazelcast-jet-core/src/test/java/com/hazelcast/jet/impl/deployment/AbstractDeploymentTest.java

```diff
@@ -323,6 +328,23 @@ public void testDeployment_whenFileAddedAsResource_thenAvailableOnClassLoader()
         executeAndPeel(getJetInstance().newJob(dag, jobConfig));
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testDeployment_whenAddSerializer_thenAvailableOnJobContext() throws Throwable {
```

Comment:
We should not test the `addClass` functionality, but that the Jet can actually use the serializer. The test could have 2 vertices connected by a distributed edge. The source will generate a non-serializable object for which a custom serializer is used. The cluster must have at least 2 members, otherwise serialization isn't used.

Reply:
I'd just remove this test. It only tests job class loading.